aasrp/templates/aasrp/partials/request_srp/fleet-details.html

```diff
@@ -10,15 +10,15 @@
 
         <div class="panel-body">
             <ul class="list-unstyled">
-                <li><b>{% translate "Fleet name" %}:</b> {{ srp_link }}</li>
+                <li><b>{% translate "Fleet Name" %}:</b> {{ srp_link }}</li>
                 <li><b>{% translate "SRP Code" %}:</b> {{ srp_link.srp_code }}</li>
 
                 {% if srp_link.fleet_type %}
                     <li><b>{% translate "Fleet Type" %}:</b> {{ srp_link.fleet_type }}</li>
                 {% endif %}
 
                 {% if srp_link.fleet_commander %}
-                    <li><b>{% translate "FC" %}:</b> {{ srp_link.fleet_commander }}</li>
+                    <li><b>{% translate "Fleet Commander" %}:</b> {{ srp_link.fleet_commander }}</li>
```

Comment:
FC is a common enough abbreviation and totally fine in this little info panel.

![image](https://user-images.githubusercontent.com/2989985/232432420-848ee8ce-6242-4ea0-981e-e766c0078c50.png)

Reply:
In such case it's required to add comment 'Keep it short' for translators for labels in this little info panel.

Reply:
Sure. This can be done by adding a `context` to the translation tag, like this:

```django
{% translate "foobar" context "Contextual Information" %}
```

In this case I'd add "Keep it an abbreviation"

Reply:
But in general, when the origin is already an abbreviation, it should also be used in translations as such.